### PR TITLE
Add OWNERS, kubernetes project template files

### PR DESCRIPTION
**Pull request by ahmetb**
https://github.com/kubernetes/kubernetes-template-project

/cc @juanvallejo @nikhita

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
-Guidelines](https://opensource.google.com/conduct/).
\ No newline at end of file
+- [Mentoring Initiatives](https://git.k8s.io/community/mentoring) - We have a diverse set of mentorship programs available that are always looking for volunteers!
+
+<!---
+Custom Information - if you're copying this template for the first time you can add custom content here, for example:
+
+## Contact Information
+
+- [Slack channel](https://kubernetes.slack.com/messages/kubernetes-users) - Replace `kubernetes-users` with your slack channel string, this will send users directly to your channel.
```
Comment (nikhita) [inline CONTRIBUTING.md]: Maybe it would be good to add the sig-cli channel and mailing list here? :)

Comment (ahmetb) [reply CONTRIBUTING.md]: Done. I reckon we'll add way more stuff about SIG CLI post-migration.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,4 @@
+aliases:
+  krew-index-maintainers:
```
Comment (rlenferink) [inline OWNERS_ALIASES]: I think the `krew-maintainers` and `krew-index-maintainers` were swapped by accident. 

Probably this one should be `krew-maintainers` and this one [8888595ca25a7a48c303fbc07f369b2db9b95268/OWNERS_ALIASES#L2](https://github.com/GoogleContainerTools/krew-index/blob/8888595ca25a7a48c303fbc07f369b2db9b95268/OWNERS_ALIASES#L2) should be `krew-index-maintainers`

Comment (ahmetb) [reply OWNERS_ALIASES]: Argh my bad. I think I’ll do both krew-maintainers if that makes it easier. I suspect we’ll revise them in the future again.

Comment (rlenferink) [reply OWNERS_ALIASES]: At the moment, it doesn't really matter. The teams in the OWNERS files need to be defined in the OWNERS_ALIASES file _per repository_. Till the moment we can use GitHub teams for this, it is necessary to have the data available in the OWNERS* files for both repositories.

So it is completely up to you if you want to 'combine' or keep the repositories separate (different contributors/maintainers).

**Pull request merged**